sql/monthly-revenue-query.sql

```diff
@@ -161,14 +161,23 @@ with conversions as (select
     CASE
         WHEN (fc.type ilike 'organization') THEN 1
         ELSE 0
-    END as "isOrg"
+    END as "isOrg",
+
+    CASE
+        WHEN (t."RefundTransactionId" is not null) THEN 1
```

Comment:
Won't this detect every refund transaction instead of only the new ones? So past months will change...

Reply:
Yeah, this was just an initial experiment to show a column with something. I've pushed another change with some more well defined filters.